### PR TITLE
ARUHA-96 Use snake_case instead of camleCase in the API

### PR DESCRIPTION
**Pull request by tmuehl**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/zalando/nakadi/pull/107" target="_blank">ReviewNinja

**Comment by v-stepanov** [comment]
EchoControllerAT test is failing :-D

**Comment by tmuehl** [comment]
@v-stepanov I know. That's why I was not asking for review to far. :)

**Comment by v-stepanov** [comment]
@tmuehl oh, sorry :)

**Comment by tmuehl** [comment]
@v-stepanov @CyberDem0n @rcillo @jkliff Please review

File: src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/PartitionsControllerAT.java [hunk 1]
```diff
@@ -141,22 +141,22 @@ public void whenGetPartitionAndWriteMessageThenOffsetInPartitionIsIncreased() th
...
+        final long offsetAfter = Long.parseLong(partitionInfoAfter.get("newest_available_offset"));
         assertThat(offsetAfter, is(offsetBefore + delta));
     }
 
     private void writeMessageToPartition(final int partition) throws InterruptedException, ExecutionException {
         final KafkaProducer<String, String> producer = kafkaHelper.createProducer();
-        final ProducerRecord<String, String> producerRecord = new ProducerRecord<>(TEST_TOPIC, partition, "blahKey",
-                "blahValue");
+        final ProducerRecord<String, String> producerRecord = new ProducerRecord<>(TEST_TOPIC, partition, "blah_key",
+                "blah_value");
```
Comment (v-stepanov) [inline src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/PartitionsControllerAT.java]: you overworked here a little bit :)

File: src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java [hunk 2]
```diff
@@ -34,4 +46,25 @@ public FilterRegistrationBean flowIdRequestFilter() {
...
+    @Bean
+    public MappingJackson2HttpMessageConverter mappingJackson2HttpMessageConverter() {
+        final MappingJackson2HttpMessageConverter converter = new MappingJackson2HttpMessageConverter();
+        converter.setObjectMapper(nakadiConfig.jacksonObjectMapper());
+        return converter;
+    }
+
+    @Override
+    protected void configureMessageConverters(final List<HttpMessageConverter<?>> converters) {
+        StringHttpMessageConverter stringConverter = new StringHttpMessageConverter();
```
Comment (CyberDem0n) [inline src/main/java/de/zalando/aruha/nakadi/config/WebConfig.java]: final

File: src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/PartitionsControllerAT.java (hunk 1 above)
Comment (tmuehl) [inline src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/PartitionsControllerAT.java]: Oh, that's an accident that this was committed :smile:

**Comment by rcillo** [comment]
I don't completely understand what's the role of each converter, but by now, if it works, :shipit:

**Pull request merged**